### ISSUE TITLE
Promise rejection with Cordova

### ISSUE THREAD

**Issue by FredGr** [OnsenUI/OnsenUI#2430]
I had the following error running an app build with cordova (browser) (see below)
It looks like it is linked with the tabbar when there are more than one tab.
It is very easy to reproduce because the same bug happens on Onsen UI tabbar live tutorial. (try to click on the second tab and F12 on Chrome).

A fast answer and correction would be highly appreciated because my application is not working anymore.

Thanks

onsenui.min.js:formatted:15199 Uncaught TypeError: Cannot read property 'promise' of undefined
    at onsenui.min.js:formatted:15199
    at Object.ready (onsenui.min.js:formatted:14688)
    at onsenui.min.js:formatted:15186
    at onsenui.min.js:formatted:13
    at Array.forEach (<anonymous>)
    at e (onsenui.min.js:formatted:12)
    at n (onsenui.min.js:formatted:29)
    at HTMLElement.value (onsenui.min.js:formatted:15175)
    at t.value (onsenui.min.js:formatted:4604)
    at t.value (onsenui.min.js:formatted:4596)
(anonymous) @ onsenui.min.js:formatted:15199
ready @ onsenui.min.js:formatted:14688
(anonymous) @ onsenui.min.js:formatted:15186
(anonymous) @ onsenui.min.js:formatted:13
e @ onsenui.min.js:formatted:12
n @ onsenui.min.js:formatted:29
value @ onsenui.min.js:formatted:15175
value @ onsenui.min.js:formatted:4604
value @ onsenui.min.js:formatted:4596
value @ onsenui.min.js:formatted:4527
(anonymous) @ onsenui.min.js:formatted:4987
value @ onsenui.min.js:formatted:14844
(anonymous) @ onsenui.min.js:formatted:14699
(anonymous) @ onsenui.min.js:formatted:13
e @ onsenui.min.js:formatted:12
n @ onsenui.min.js:formatted:29
e @ onsenui.min.js:formatted:14698
value @ onsenui.min.js:formatted:4579
value @ onsenui.min.js:formatted:4527
(anonymous) @ onsenui.min.js:formatted:4972
value @ onsenui.min.js:formatted:11313
(anonymous) @ onsenui.min.js:formatted:11290
(anonymous) @ onsenui.min.js:formatted:13
e @ onsenui.min.js:formatted:12
n @ onsenui.min.js:formatted:29
e @ onsenui.min.js:formatted:11289
value @ onsenui.min.js:formatted:4579
value @ onsenui.min.js:formatted:4567
value @ onsenui.min.js:formatted:4650
onsenui.min.js:formatted:15199 Uncaught TypeError: Cannot read property 'promise' of undefined
    at onsenui.min.js:formatted:15199
    at Object.ready (onsenui.min.js:formatted:14688)
    at onsenui.min.js:formatted:15186
    at onsenui.min.js:formatted:13
    at Array.forEach (<anonymous>)
    at e (onsenui.min.js:formatted:12)
    at n (onsenui.min.js:formatted:29)
    at HTMLElement.value (onsenui.min.js:formatted:15175)
    at t.value (onsenui.min.js:formatted:4604)
    at t.value (onsenui.min.js:formatted:4596)

The line of code is : 
                                return t.isActive() ? s() : i._loadInactive.promise.then(s)

**Comment by asialgearoid** [comment]
Thanks for the report @FredGr. It looks like this was caused by a fix for https://github.com/OnsenUI/OnsenUI/issues/2324. It seems to have caused a race condition with the loading of tabs, so it only happens in some cases, which is why we did not come across it in our testing.

I have fixed it now. I will wait to see if there are any other bug reports from the release of 2.10.0, and will push out 2.10.1 next week with this fix and any others that may come along. Until then, I recommend going back to 2.9.2.
